jade/enums.py

```diff
@@ -19,8 +19,16 @@ class Mode(enum.Enum):
     SEQUENTIAL = "sequential"
 
 
+class JobCompletionStatus(enum.Enum):
```

Comment:
We were using free-form strings everywhere and it was confusing. This should make it more robust.